### PR TITLE
PLENTY-313 Move the filter image check logic to the frontend

### PR DESCRIPTION
**Pull request by RimantasAdeoWeb**

### CODE REVIEW

File: resources/views/ItemList/Components/Filter/ItemColorTiles.twig [hunk 1]
```diff
@@ -17,4 +21,4 @@
             </li>
         </ul>
     </div>
-</script>
+</script>
```
Comment (TheKeymaster) [inline resources/views/ItemList/Components/Filter/ItemColorTiles.twig:24]: Please add a newline at the end :)

File: resources/views/ItemList/Components/Filter/ItemFilterImage.twig [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                <label :for="'option-' + value.id" class="form-check-label" rel="nofollow">
+                    <img :src="value.imageUrl" @error="handleImageError($event)" width="80px"/>
+                    <span v-text="value.name"></span>
+                </label>
+                <div class="filter-badge" v-if="value.count" v-text="value.count"></div>
+            </div>
+        </div>
+        <p v-if="facet.noAvailableFiltersText" v-text="facet.noAvailableFiltersText"></p>
+    </div>
+</script>
```
Comment (TheKeymaster) [inline resources/views/ItemList/Components/Filter/ItemFilterImage.twig:15]: Newline :).

File: resources/views/ItemList/Components/Filter/ItemFilterImage.twig [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<script type="x/template" id="vue-findologic-item-filter-image">
+    <div>
+        <div v-if="!facet.noAvailableFiltersText">
+            <div class="form-check" v-for="value in facet.values" :key="value.id">
+                <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" @change="updateFacet(value)" :disabled="isLoading">
+                <label :for="'option-' + value.id" class="form-check-label" rel="nofollow">
+                    <img :src="value.imageUrl" @error="handleImageError($event)" width="80px"/>
```
Comment (TheKeymaster) [inline resources/views/ItemList/Components/Filter/ItemFilterImage.twig:7]: Nice!

File: tests/Api/Response/Parser/FiltersParserTest.php [hunk 4]
```diff
@@ -346,6 +346,7 @@ public function parseDataProvider()
                                 'position' => 'item',
                                 'count' => '25',
                                 'image' => 'https://test.com/vendor/a_amp_c_design.jpg',
```
Comment (TheKeymaster) [inline tests/Api/Response/Parser/FiltersParserTest.php:348]: Why do we have an image and an image URL? Isn't this the same?

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: tests/Api/Response/Parser/FiltersParserTest.php (hunk 4 above)
Comment (RimantasAdeoWeb) [reply tests/Api/Response/Parser/FiltersParserTest.php:348]: The difference is that imageUrl doesn't always get added, depending on what sort of value the image is. I wouldn't know all the details on how and why it's done like this though.

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**